### PR TITLE
Wallabag : adds an option to send the review as tags on wallabag

### PR DESCRIPTION
**Pull request by agallou**
This will allow koreader users to choose to add tags in the review field (separated by comas), and then send them to wallbag.
It adds and option for that (a checkbox) in the wallabag plugin options.
It could be useful to directly tag articles on koreader to find them easily later.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/8637)

### CODE REVIEW

File: plugins/wallabag.koplugin/main.lua [hunk 1]
```diff
@@ -258,6 +262,17 @@ function Wallabag:addToMainMenu(menu_items)
                             },
                         },
                     },
+                    {
+                        text = _("Send review as tags"),
```
Comment (Frenzie) [inline plugins/wallabag.koplugin/main.lua:266]: I think a little bit more explanation would be helpful. :-) Something like this:
```suggestion
                        text = _("Send review as tags"),
                        help_text = _("This allow you to write tags in the review field, separated by commas, which can then be sent to Wallabag."),
```

**Review by Frenzie — COMMENTED** [review]
Btw, if you rebase on top of master the CI will run instead of showing unauthorized. Looks great though, I doubt luacheck will have any complaints. ^_^

**Comment by agallou** [comment]
@Frenzie thanks, I've just added your help text and rebased the branch.

File: plugins/wallabag.koplugin/main.lua [hunk 2]
```diff
@@ -780,6 +801,39 @@ function Wallabag:addArticle(article_url)
...
+            local headers = {
+                ["Content-type"] = "application/json",
+                ["Accept"] = "application/json, */*",
+                ["Content-Length"] = tostring(#bodyJSON),
+                ["Authorization"] = "Bearer " .. self.access_token,
+            }
+
+            self:callAPI("POST", "/api/entries/" .. id .. "/tags.json", headers, bodyJSON, "")
+        else
+          logger.dbg("Wallabag: no tags to send for ", path)
```
Comment (Frenzie) [inline plugins/wallabag.koplugin/main.lua:831]: Odd, it still says unauthorized. Oh well, no matter. Btw, one tiny nit but I'll just add it through the GH web UI. :-)
```suggestion
            logger.dbg("Wallabag: no tags to send for ", path)
```

**Comment by Frenzie** [comment]
Thanks!

**Pull request merged**